tests/unit/test_tf_feature_columns.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+tf = pytest.importorskip("tensorflow")
+nvtf = pytest.importorskip("nvtabular.framework_utils.tensorflow")
+
+
+def test_feature_column_utils():
```

Comment:
nice! I like having a test here - the feature column utils was a notable gap in our code coverage